types/route-cache/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for route-cache 0.5
+// Project: https://github.com/bradoyler/route-cache
+// Definitions by: yutak23 <https://github.com/yutak23>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as express from 'express';
+
+export function config(options: ConfigOptions): RouteCache;
+export function cacheSeconds(secondsTTL: number, cacheKey: string | CacheKeyFunc): express.RequestHandler;
+export function removeCache(url: string): void;
+
+export const cacheStore: Store;
+
+export interface RouteCache {
+    config(options: ConfigOptions): RouteCache;
+    cacheSeconds(secondsTTL: number, cacheKey: string | CacheKeyFunc): express.RequestHandler;
+    removeCache(url: string): void;
+    cacheStore: Store;
+}
+
+export interface CacheKeyFunc {
+    (req: express.Request, res: express.Response): string | null;
+}
+
+export interface ConfigOptions {
+    max?: number;
+    cacheStore?: Store;
+}
+
+export class Store {
```

Comment:
This shouldn't be exported as a class, because it implies there is a run time export called 'Store' that is a class value, when in fact there isn't. You can export it as an interface instead, or just keep the class without exporting it.

Reply:
thank you for review and comment.
I have modified the interface to export according to your suggestion.